var/spack/repos/builtin/packages/libzip/package.py

```diff
@@ -51,9 +54,36 @@ def url_for_version(self, version):
         conditional("cmake", when="@1.4:"), conditional("autotools", when="@:1.3"), default="cmake"
     )
 
+    # Required dependencies
+    with when("build_system=cmake"):
+        variant("gnutls", default=True, description="Enable gnutls support")
+        variant("bz2", default=True, description="Enable bzip2 support")
```

Comment:
Any reason this isn't `bzip2` since that's what the CMake option uses and what the dependency package is called? Is there a preference for bz2 variant name over bzip2?

Reply:
I don't know , I found this in other packages so I used it but bzip2 is fine by me

Reply:
I renamed the variant bzip2, I also found an extra space after GNUTLS